yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -262,6 +273,11 @@ func createDBConfigFromRecord(record []string, header []string) (*AssessMigratio
 		return nil, fmt.Errorf("mandatory fields missing in the record: '%s'", strings.Join(missingFields, "', '"))
 	}
 
+	// Check if the source-db-type is supported (only 'oracle' allowed)
+	if dbType, ok := configMap[SOURCE_DB_TYPE]; ok && strings.ToLower(dbType) != ORACLE {
```

Comment:
dbtype is a mandatory argument, and it is checked above. You need not check with `_, ok` syntax.